shortlist/solutions/C1.md

```diff
@@ -0,0 +1,16 @@
+## Proof
+
+We tile the rectangle as so:
+[Figure 1] (where the corners are black)
+
+Consequently, there is at least 1 rectangle with odd side length and black corners. (**Lemma 1**)
+
+### Proof of Lemma 1:
+
+Assume the contrary for the sake of contradiction. Then there are only rectangles with even side lengths, which have an equal amount of white and black squares, or rectangles with odd side lengths and white corners, which have more white squares than black squares. However, there has to be more black squares than white squares, contradiction.
+
+By **Lemma 1**, we need to prove that the odd rectangle satisfies those restrictions. We do this by numbering the rectangle like so:
+
+[Figure 2]
```

Comment:
What you can do to formalize the statements in the diagrams is something like:

Number all the 1x1 squares in the oddxodd rectangle in lexicographic order, so the top row is 1, 2, 3, 4, ..., w, the next row is w+1, w+2, w+3, ... 2w ... and the last row is (h-1)w+1, (h-1)w+2, ... hw.

Because w is odd it is necessarily the case that kw and kw+1 are opposite colors for all k, and thus the entire coloring of the sequence from 1 to hw is perfectly alternating black, white, black, white, ... black.
This leaves no doubt that the number of black squares is one more than the number of white squares, and a rectangle in the partition has 4 black corners iff the indices of the corners are bw+a, bw+c, dw+a, and dw+c for values of a, b, c, and d where the values of a-c and b-d are both even.